builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -128,11 +128,6 @@ steps:
   workingDirectory: $(Build.SourcesDirectory)/samples/samples-python
   displayName: Lint samples-python
 
-- script: |
-    pip3 install -r requirements.txt
```

Comment:
Don't we still need to install the azure-functions library? It just won't be pinned anymore (so you'd just update this to have a name like `Install samples-python dependencies`)